frontend/js/instructions.js

```diff
@@ -0,0 +1,74 @@
+const renderInstructionsHtml = (params) => {
+  return (
+    `<body>
+        <header class="main-header" onLoad="CallbackFunction();">
```

Comment:
What is CallbackFunction() here?

Reply:
This function was added previously for some testing purpose. It is removed now from the code